dht.go

```diff
@@ -84,15 +84,17 @@ type Config struct {
 	// MaxInfoHashPeers is the limit of number of peers to be tracked for each infohash. A
 	// single peer contact typically consumes 6 bytes. Default value: 256.
 	MaxInfoHashPeers int
+	//Protocol for UDP connections, udp4= IPv4, udp6 = IPv6
+	UDPProto string
 }
 
 // Creates a *Config populated with default values.
 func NewConfig() *Config {
 	return &Config{
 		Address:          "",
-		Port:             0, // Picks a random port.
+		Port:             8444, // Picks a random port.
 		NumTargetPeers:   5,
-		DHTRouters:       "de.magnets.im:6881,cz.magnets.im:6881,router.bittorrent.com:6881,dht.transmissionbt.com:6881",
+		DHTRouters:       "[2001:41d0:1:aa81:2::1]:6881",
```

Comment:
The IPv6 IP for the bootstrap node was dht.transmissionbt.com which has IPv4 and IPv6 (A and AAAA) records.  I returned the list to the popular bootstrap nodes.